labman/gui/static/js/plateViewer.js

```diff
@@ -221,9 +263,7 @@ PlateViewer.prototype.initialize = function (rows, cols) {
       return;
     }
     var row = $(this).data("row");
-    // col - 1 to get the correct index, since column 0 is the header
-    // in the actual slickgrid
-    var col = $(this).data("col") - 1;
+    var col = $(this).data("col");
```

Comment:
The -1 isn't needed anymore?

Reply:
The indices refer to the first column, the one that used to have the row names (A, B, C ...). Since the row names have now been moved to a new table the `- 1` is no longer needed.